activerecord/test/schema/schema.rb

```diff
@@ -252,6 +252,16 @@
     t.string :status
   end
 
+  create_table :travel_routes, primary_key: [:origin, :destination], force: true do |t|
```

Comment:
Do we need this new table or is it possible to use an existing one? We try not to add new tables if we can use an existing one.

Reply:
Not a necessity, we definitely can use existing `cpk_books` table/model to test the `dup` behavior. I just find `travel_routes` to be very friendly to a reader.
I'll update the PR to use `cpk_books` for now and consider adding `travel_routes` when we have an actual use-case

Reply:
Done, removed the new model in favor of an existing one